modules/gob-examine-student/source/count-courses.ts

```diff
@@ -10,7 +9,7 @@ import type { Course } from "./types"
  * @private
  * @param {Course[]} courses - the list of courses
  * @returns {number} - the number of unique courses
- */
+,
```

Comment:
```suggestion
 */
```

There are so many of these invalid block comments!